src/client/js/listing.js

```diff
@@ -16,25 +16,24 @@
  *  limitations under the License
  *
  */
- (function() {
-  let database = firebase.database();
-  firebase.database().ref('/products/').once('value').then(function(snapshot) {
+export default function initializeProducts() {
+  firebase.database().ref('/products/').once('value').then((snapshot) => {
     populatePiesList(snapshot.val());
   });
 
   function populatePiesList(pies) {
-    let pieList = document.querySelector('#pies');
```

Comment:
BTW: some pedants may grumble that `document.querySelector('#pies')` is slower than `document.getElementById('pies')`. 

Personally, I think performance is irrelevant here, and it can be clearer to use `document.querySelector()` everywhere, but just in case someone whinges...

Reply:
I prefer the flexibility and consistency of `querySelector()`. Perhaps if this were inside of a `for` loop that could be a concern, but even then I wouldn't make any changes before profiling and confirming it was indeed an issue.